Makefile

```diff
@@ -59,7 +65,7 @@ up: ## Runs the local container(s) (n=service name, p=profile name)
 
 down: ## Stops the local containers and removes them
 	$(info Stops the local containers and removes them)
-	@docker-compose --profile all down
+	@docker-compose --profile all down -v
```

Comment:
Get rid of volumes when using this method.  Not sure if this is ideal, but this is what I want normally when I use this command.